recipes/google-auth/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "google-auth" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "dd7b6e5f3bd15c0f7cfee63a266acea40c779da0a924b475bc1812e6e10687e1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pyasn1 >=0.1.7,<1dev
+    - pyasn1-modules >=0.0.5,<1dev
+    - rsa >=3.1.4,<4dev
+    - six >=1.9.0,<2dev
+    - cachetools >=2.0.0,<3dev
+
+test:
+  imports:
+    - google.auth
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-auth-library-python
+  license: Apache
```

Comment:
Please add the license version. Think it is 2.0. Also please add `license_file: LICENSE` too so that packages will contain the license file from the source code.